### PR TITLE
initial joss draft

### PR DESCRIPTION
**Pull request by mgxd**
[rendered](https://res.cloudinary.com/hju22ue2k/image/upload/v1573588943/hmule6phqyl9pzwxiqnr.pdf)

### CODE REVIEW

File: joss/paper.md [hunk 1]
```diff
@@ -0,0 +1,118 @@
...
+ - name: Department of Otolaryngology, Harvard Medical School, Boston, MA, USA
+   index: 3
+date: 04 November 2019
+bibliography: nt.bib
+---
+
+# Summary
+
+Spatial transforms formalize mappings between coordinates of objects in
+biomedical images. Transforms typically are the outcome of image registration
```
Comment (oesteban) [inline joss/paper.md]: Please start new sentences with new lines:
```suggestion
biomedical images.
Transforms typically are the outcome of image registration
```

File: joss/paper.md [hunk 2]
```diff
@@ -0,0 +1,118 @@
...
+registration is a prominent task present in image processing. In neuroimaging,
+the proliferation of image registration software implementations has resulted
+in a disparate collection of structures and file formats used to preserve and
+communicate the transformation. This assortment of formats presents the
+challenge of compatibility between tools and endangers the reproducibility of
+results.
+
+`nitransforms` is a Python tool capable of reading and writing tranforms
+produced by the most popular neuroimaging software (AFNI [-@cox_software_1997],
+FSL [-@jenkinson_fsl_2012], FreeSurfer [-@fischl_freesurfer_2012], ITK, and SPM).
```
Comment (oesteban) [inline joss/paper.md]:
```suggestion
FSL [-@jenkinson_fsl_2012], FreeSurfer [-@fischl_freesurfer_2012],
ITK via ANTs [-@avants_symmetric_2008], and SPM [-@friston_statistical_2006]).
```

with:

```Bibtex

@article{avants_symmetric_2008,
	author = {Avants, B.B. and Epstein, C.L. and Grossman, M. and Gee, J.C.},
	doi = {10.1016/j.media.2007.06.004},
	issn = {1361-8415},
	journal = {Medical Image Analysis},
	number = 1,
	pages = {26-41},
	shorttitle = {Symmetric diffeomorphic image registration with cross-correlation},
	title = {Symmetric diffeomorphic image registration with cross-correlation: Evaluating automated labeling of elderly and neurodegenerative brain},
	url = {http://www.sciencedirect.com/science/article/pii/S1361841507000606},
	volume = 12,
	year = 2008
}

@book{friston_statistical_2006,
	address = {London},
	author = {Friston, Karl J. and Ashburner, John and Kiebel, Stefan J. and Nichols, Thomas E. and Penny, William D.},
	isbn = {978-0-12-372560-8},
	publisher = {Academic Press},
	shorttitle = {Statistical parametric mapping},
	title = {Statistical parametric mapping : the analysis of functional brain images},
	year = 2006
}
```

File: joss/paper.md [hunk 3]
```diff
@@ -0,0 +1,118 @@
...
+Additionally, the tool provides seamless conversion between these formats, as
+well as the ability of applying the transforms to other images. `nitransforms`
```
Comment (oesteban) [inline joss/paper.md]: Better in italics (IMHO)
```suggestion
well as the ability of applying the transforms to other images.
*NiTransforms* is inspired by *NiBabel* [@brett_nibabel_2006], a Python package with a
```

File: joss/paper.md [hunk 4]
```diff
@@ -0,0 +1,118 @@
...
+image fusion application would be the alignment of functional data from one
+individual's brain to the same individual's corresponding anatomical MRI scan
+for visualization. Therefore, "applying a transform" entails two operations:
+first, transforming the coordinates of the samples in the reference image $R$ to
+find their mapping $\vec{x}'$ on $M$ via $T\{\cdot\}$, and second, an
+interpolation step as $\vec{x}'$ will likely fall off-the-grid of the moving
+image $M$. These two operations are confusing because, while the spatial
+transformation projects from $R$ to $M$, the data flows in reversed way after the
+interpolation of the values of $M$ at the mapped coordinates $\vec{x}'$.
+
```
Comment (oesteban) [inline joss/paper.md]: I'd add a section of examples, copying from the preprint.

Comment (mgxd) [reply joss/paper.md]: their docs explicitly state
> software documentation such as API (Application Programming Interface) functionality should not be in the paper and instead should be outlined in the software documentation.

Comment (oesteban) [reply joss/paper.md]: I think an example or two cannot be considered software documentation, and they will very quickly convey what the tool does (ideally, I'm imagining two images on the left with some object signifying an input transform, they feed into the snippet and the output shows the moving image deformed via T). I can take a stab on this next week, don't worry about this.

**Pull request merged**